src/__stories__/logo-story.tsx

```diff
@@ -80,7 +81,7 @@ Default.args = {
 
 Default.argTypes = {
     brand: {
-        options: ['Movistar', 'O2', 'Vivo', 'Telefonica', 'Blau'],
+        options: ['Movistar', 'O2', 'Vivo', 'Telefonica', 'Blau', 'Tu'],
```

Comment:
Should we add a screenshot test for this? ([in this line](https://github.com/Telefonica/mistica-web/blob/master/src/__screenshot_tests__/logo-screenshot-test.tsx#L3))

Reply:
I can include it, no problem